tests/Resources/ODS.robot

```diff
@@ -139,3 +139,15 @@ RHODS Namespaces Should Not Exist
     Verify Project Does Not Exists  redhat-ods-monitoring
     Verify Project Does Not Exists  redhat-ods-applications
     Verify Project Does Not Exists  redhat-ods-operator
+
+Get Notification Email From Addon-Managed-Odh-Parameters Secret
+    [Documentation]    Gets email form addon-managed-odh-parameters secret
+    ${resp} =    Oc Get  kind=Secret  namespace=redhat-ods-operator  name=addon-managed-odh-parameters
+    ${resp} =  Evaluate  dict(${resp[0]["metadata"]["annotations"]["kubectl.kubernetes.io/last-applied-configuration"]})
+    [Return]  ${resp["stringData"]["notification-email"]}
+
+Notification Email In Alertmanager ConfigMap Should Be
+    [Documentation]    Check expected email is present in Alertmanaher
```

Comment:
Typo in Alertmanager

Reply:
typo still present, re-opining the conversation

Reply:
done